rust/rust.dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM janx/ubuntu-dev
+MAINTAINER Jan Keromnes "janx@linux.com"
+
+RUN sudo apt-get update -q \
+ && sudo apt-get upgrade -qy \
+ && sudo DEBIAN_FRONTEND=noninteractive apt-get install -qy \
+  build-essential \
+  cmake
```

Comment:
Nit: You don't need to install `build-essential` or `cmake` here, because they're [already part of ubuntu-dev](https://github.com/JanitorTechnology/dockerfiles/blob/7e68c39521aea84d43a0d3807a56857d67f6190d/ubuntu-dev/ubuntu-dev.dockerfile#L27-L31).